scripts/relauncher.py

```diff
@@ -5,7 +5,7 @@
     print('Relauncher: Launching...')
     if n > 0:
         print(f'\tRelaunch count: {n}')
-    os.system("python scripts/webui.py")
+    os.system("python -u scripts/webui.py")

```

Comment:
is unbuffered output a quirk of docker support? i can't think of any reason it will affect regular users, but i will wait to confirm before i merge

Reply:
Very possible that is the case. Without setting unbuffered I was not even seeing `Relauncher: Launching...` or any other messages in STDOUT until after I had loaded the UI and submitted something

Reply:
Hopefully you were able to confirm, I do not have a windows system to test.